pkg/build/templates/modify-raw-image.sh.tpl

```diff
@@ -12,6 +12,17 @@ set -euo pipefail
 #
 # Guestfish Command Documentation: https://libguestfs.org/guestfish.1.html
 
+# Resize the raw disk image to accommodate the users desired raw disk image size
+# This is also required if embedding content into /combustion, especially for airgap.
+# Should *only* execute if the user is building a raw disk image.
+{{ if ne .DiskSize "" -}}
+truncate -r {{.ImagePath}} {{.ImagePath}}.expanded
+truncate -s {{.DiskSize}} {{.ImagePath}}.expanded
+virt-resize --expand /dev/sda3 {{.ImagePath}} {{.ImagePath}}.expanded
```

Comment:
Can we include these in the unit test as well?

Reply:
First is in, second I just added.